artifacts/deploy/karmada-apiserver.yaml

```diff
@@ -61,7 +61,7 @@ spec:
             - --tls-cert-file=/etc/kubernetes/pki/karmada.crt
             - --tls-private-key-file=/etc/kubernetes/pki/karmada.key
           name: karmada-apiserver
-          image: k8s.gcr.io/kube-apiserver:v1.19.1
+          image: k8s.gcr.io/kube-apiserver:v1.19.11
```

Comment:
I'd prefer to use the latest patch release `v1.19.14`.

By the way, we are going to update k/k to `v1.20`, and then `v1.21`.